src/Estimators/src/SubModel.cpp

```diff
@@ -489,19 +498,73 @@ std::size_t RDE::SubModel::getNrOfExternalContact() const
     return m_externalContactList.size();
 }
 
-const RDE::FT& RDE::SubModel::getFTSensor(const int index) const
+const RDE::FT& RDE::SubModel::getFTSensor(const std::string& name)
+{
+    if (m_ftList.find(name) != m_ftList.end())
+    {
+        return m_ftList[name];
+    }
+
+    log()->error("[RobotDynamicsEstimator::getFTSensor] Sensor not found.");
```

Comment:
I would suggest adding the name of the sensor that has not been found. The same below